tests/integration/ceph_mgr_test.go

```diff
@@ -125,14 +125,22 @@ func (suite *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
 	for timeout := 0; timeout < 30; timeout++ {
 		err, output := suite.execute([]string{"status"})
-		logger.Info("%s", output)
+		logger.Infof("output: %s", output)
 		if err == nil {
 			logger.Info("Rook Toolbox ready to execute commands")
+			if !strings.Contains(output, "rook") {
```

Comment:
Is there a more specific string we can check for than rook? The module should be the only reason "rook" is found though, so maybe it's fine.

Reply:
I'm afraid but the way to know what is the active orchestrator is to take a look to the output of this command. I have converted it to a map in order to make a more "precise" verification, but in any case what we have in the output, and what we need to verify is that "rook" appears as "backend"